test/e2e/deferred/restart.go

```diff
@@ -20,6 +22,23 @@ import (
 	"github.com/openshift/cluster-node-tuning-operator/test/e2e/util/wait"
 )
 
+const (
+	enableRestartTestsEnvVar = "DEFERRED_RESTART_TESTS"
+)
+
+func areRestartTestsEnabled() bool {
```

Comment:
`Ginkgo` has a built-in Labels mechanism that allow you to skip/unskip tests in a more natural manner. 
Unless the commands the executes the tests sits somewhere else (on different code-base) then we need to actively configure it to skip the label, which is less convenient.

Reply:
very good point, let's add labels, and let's fix the matching tags